File: problemas/1000/1071.dart
import 'dart:io';
import 'dart:math';

void main() {
  final x = int.parse(stdin.readLineSync());
  final y = int.parse(stdin.readLineSync());
  final menor = min(x, y) + 1;
  final maior = max(x, y) - 1;
  int soma = 0;

  for (var i = menor; i <= maior; i++) {
    if (i % 2 != 0) {
      soma += i;
    }
  }

  stdout.writeln(soma);
}

File: problemas/1000/1071.js
const [x, y] = require('fs')
  .readFileSync('/dev/stdin', 'utf-8')
  .split('\n')
  .map(x => Number(x));

const menor = Math.min(x, y) + 1;
const maior = Math.max(x, y) - 1;
var soma = 0;

for (let i = menor; i <= maior; i++) {
  if (i % 2 !== 0) {
    soma += i;
  }
}

console.log(soma);